guides/common/modules/snip_users-to-avoid-in-foreman-deb.adoc

```diff
@@ -0,0 +1,9 @@
+[id="users-to-avoid-in-foreman-deb_{context}"]
+* www-data
+ifeval::["{context}" == "{project-context}"]
+* foreman
+endif::[]
```

Comment:
```suggestion
* foreman
```

I assume this can be shortened, because my understanding is that the build target for Foreman on Debian is "foreman-deb", same as the project-context attribute you've defined.